app/scripts/views/settings/avatar_camera.js

```diff
@@ -148,10 +148,17 @@ function (_, Cocktail, canvasToBlob, FormView, ProgressIndicator,
 
     submit: function () {
       var self = this;
+      var account = self.getSignedInAccount();
+      // if the user already has an image set, then report a change event
+      if (self._hasPreviousProfileImage(account)) {
```

Comment:
Since this is repeated 3x, you could just extract the entire thing into avatar-mixin.js `logAvatarChange` or something similar?